dev/com.ibm.ws.security.oauth/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -131,6 +131,9 @@ issueRefreshToken.desc=A value of false disables generation and the use of refre
 refreshTokenLength=Refresh token length
 refreshTokenLength.desc=Length of generated refresh token.
 
+refreshedAccessTokenLimit=Refreshed access token limit
+refreshedAccessTokenLimit.desc=Maximum number of access tokens that may be created using refresh tokens for a single OAuth client and user combination.
```

Comment:
Change,
"Maximum number of access tokens that may be created using refresh tokens for a single OAuth client and user combination."
to,
"Maximum number of access tokens that can be created by using refresh tokens for a single OAuth client and user combination."